docker-compose.yml

```diff
@@ -0,0 +1,230 @@
+version: '2'
+
+services:
+  taigadb:
+    image: postgres:10-alpine
+    container_name: taigadb
+    #restart: always
+    ports:
+      - 5432:5432
+    volumes:
+      - /srv/taiga/db/data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=taiga
+      - POSTGRES_USER=taiga
+      - POSTGRES_PASSWORD=${TAIGA_DB_PWD}
+
+  taiga_back:
+    # For CI or local modifications
+    build:
+      context: .
+      dockerfile: Dockerfile
+    # For production
+    #image: monogramm/docker-taiga-back:4.2-alpine
+    hostname: ${HOSTNAME}
+    container_name: taiga_back
+    #restart: always
+    depends_on:
+      - taigadb
+    ports:
+      - 8001:8001
+    volumes:
+      # Media and uploads directory. Required (or you will lose all uploads)
+      - /srv/taiga/back/media:/usr/src/taiga-back/media
+      - /srv/taiga/back/static:/usr/src/taiga-back/static
+      # Taiga configuration directory. Makes it easier to change configuration with your own
+      #- /srv/taiga/back/conf:/taiga
+      - /usr/local/lib/python3.6/site-packages/taiga_contrib_ldap_auth_ext
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+    environment:
+      # Your hostname (REQUIRED)
+      - TAIGA_HOSTNAME=${HOSTNAME}:${PORT}
+      #- TAIGA_SSL=False
+      #- TAIGA_SSL_BY_REVERSE_PROXY=True
+      # Secret key for cryptographic signing
+      - TAIGA_SECRET_KEY=${TAIGA_SECRET}
+      # Admin account default password
+      - TAIGA_ADMIN_PASSWORD=${TAIGA_ADMIN_PASSWORD}
+      # Database settings
+      - TAIGA_DB_HOST=taigadb
+      - TAIGA_DB_NAME=taiga
+      - TAIGA_DB_USER=taiga
+      - TAIGA_DB_PASSWORD=${TAIGA_DB_PWD}
+      # when the db comes up from docker, it is usually too quick
+      - TAIGA_SLEEP=5
+      # To use an external SMTP for emails, fill in these values:
+      - TAIGA_ENABLE_EMAIL=True
+      - TAIGA_EMAIL_FROM=taiga@${TAIGA_EMAIL_DOMAIN}
+      - TAIGA_EMAIL_USE_TLS=True
+      - TAIGA_EMAIL_HOST=smtp.${TAIGA_EMAIL_DOMAIN}
+      - TAIGA_EMAIL_PORT=587
+      - TAIGA_EMAIL_USER=${TAIGA_SMTP_USER}
+      - TAIGA_EMAIL_PASS=${TAIGA_SMTP_PWD}
```

Comment:
@AminovE99 Setup this to use a local [mailcatcher](https://github.com/sj26/mailcatcher) instance.